helpers.py

```diff
@@ -83,4 +84,99 @@ def graph_one(tr, pr, quantiles,  env_name, dataset_name, m=MaxNLocator, framewo
         os.makedirs('log/fig')
     plt.savefig(filename)
     # plt.show()
-    plt.close()	
\ No newline at end of file
+    plt.close()
+
+
+def plot_seeds(tr, pr, quantiles, env_name, dataset_name, m=MaxNLocator, framework='keras', width=.35, title=""):
```

Comment:
this seems like it was a useful tool for you. It's not core code, so I won't examine it carefully